spec/lib/nexmo_rails_spec.rb

```diff
@@ -25,5 +26,48 @@
       expect(client.private_key).to eq('NEXMO_PRIVATE_KEY')
     end
   end
+
+  describe 'private key initialization with file path defined' do
+
+    before do 
+      allow(File).to receive(:exists?).and_return(true)
```

Comment:
@benhayehudi is the idea behind this spec to test that it loads the content from the file?

Reply:
Yes, correct. Depending on the value provided, this is a check of the logic provided for private_key.

Reply:
In that case, I think it would be best if we add a file to the dummy app and use that one for testing WDYT? I can give you a hand with that

Reply:
disregard, it's testing a different flow